crates/rspack_core/src/exports_info.rs

```diff
@@ -1501,19 +1507,9 @@ impl ExportInfo {
     }
   }
 
-  fn get_max_target_readonly(&self) -> &HashMap<Option<DependencyId>, ExportInfoTargetValue> {
-    assert!(self.max_target_is_set);
-    &self.max_target
-  }
-
-  fn get_max_target(&mut self) -> &HashMap<Option<DependencyId>, ExportInfoTargetValue> {
```

Comment:
Yes, so I do not change the logic of get_max_target(), and only add _get_max_target() calling [here](https://github.com/web-infra-dev/rspack/blob/refactor/immutable-module-graph/crates/rspack_core/src/exports_info.rs#L2047) to make sure it can be generated even when module graph is immutable